_sass/layout/_header.scss

```
@@ -2,66 +2,168 @@
 @use "../scss/config" as *;
 @use "../vendors/include-media/include-media" as *;
 
-.home {
-  &-header {
-    margin: 20% 0 2em;
+.pre-header {
+  background-color: $darkred;
+  color: $white;
 
-    @include media(">medium") {
-      margin: 15% 0 2em;
-    }
-    @include media(">large") {
-      margin: 10% 0 2em;
-    }
-    @include media(">xlarge") {
-      margin: 8% 0 2em;
+  & .wrapper{
+    display: flex;
+    padding-top: .66em;
+    padding-bottom: .66em;
+    align-items: center;
+    justify-content: center;
+    gap: 3.2rem;
+    align-self: stretch;
+    flex-direction: row;
+  }
+
+  @include media(">=large") {
+    & .wrapper{
+      justify-content: space-between;
     }
   }
 
-  &-main-title {
-    font: $title-font-style $title-font-weight calc(1em / $default-line-height)
-      $title-font-family;
+  &-tagline {
+    font-weight: 600;
+    line-height: 150%;
     text-align: center;
-    color: inherit;
+  }
+}
 
-    // &-logo {
-    //   width: 100%; height: auto;
-    //   font: $logo-font-style $logo-font-weight 1.6rem/1 $logo-font-family;
+.top {
+  &-header {
+    position: relative;
+    overflow: hidden;
+    color: $white;
+
+    & > img {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
 
-    //   image {
-    //     position: fixed;
-    //   }
+    & > .heart {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: auto;
+      height: 100%;
+      padding: 3.2rem 0;
+      box-sizing: border-box;
+    }
 
-    //   text {
-    //     fill: url(#pic);
-    //   }
-    // }
+    &-buttons {
+      margin: 3.6rem 0;
+      display: flex;
+      gap: 1.6rem;
+      align-items: center;
+    }
+
+    & .wrapper {
+      position: relative;
+      z-index: 1;
+      padding: 3.2rem;
```

```suggestion
      padding: 3.2rem $half-gutter;
```

Petit détail mais si on change les tailles du half-gutter, je pense que ça marchera mieux ainsi.